packages/solo/src/chain-cosmos-sdk.js

```diff
@@ -759,6 +760,7 @@ ${chainID} chain does not yet know of address ${clientAddr}${adviseEgress(
     // Reset the backoff period.
     retryBackoff = randomizeDelay(INITIAL_SEND_RETRY_DELAY_MS);
   };
+  /** @param {Error} [e]  */
```

Comment:
The special thing going on is `= (e = undefined) => {` which gives a type hint to TS so that `e` is not any, it's `undefined`. Then passing errors to `failedSend` doesn't satisfy the constraint. This annotation indicates it can be Error _or_  undefined